prometheus/benchmark_test.go

```diff
@@ -56,6 +79,37 @@ func BenchmarkCounterWithLabelValuesConcurrent(b *testing.B) {
 	wg.Wait()
 }
 
+func BenchmarkCounterWithLabelValuesAndConstraintsConcurrent(b *testing.B) {
```

Comment:
Those feel duplicated below, no? Or at least the difference is small.

Reply:
It's indeed a small difference, there's a constraint whereas the first version doesn't have any constraint at all. It lets us assess the impact of constraints on performances and allocations.